README.md

```diff
@@ -103,48 +103,52 @@ int main(int argc, char **argv)
 TinySpline uses the CMake build system to compile and package the interfaces. The C
-library has been implemented in C89 and, thus, should be compilable with nearly every
+library has been implemented in C89 and, thus, should be compatible with every modern
 compiler. All other features of TinySpline are optional and will be disabled if CMake does
-not find the required dependencies. However, CMake and an appropriate C/C++ compiler must
-be available, regardless of the interface you want to build. TinySpline supports any of
+not find the required dependencies; however, CMake and an appropriate C/C++ compiler must
+be available, regardless of the interface you want to build. TinySpline is tested using
 the following compiler suites: gcc, clang, and msvc. In order to compile the C++ wrapper
 or any of the bindings, your compiler suite (e.g. gcc) must support C++11. Additionally,
-Swig (in version 3.0.1 or above) is required to generate the bindings. Along Swig, each
-binding may have further dependencies to generate the source code of the target language.
+Swig (in version 3.0.1 or above) is required to generate the bindings for other languages.
+Each binding may have further dependencies to generate the source code of the target language.
 The following table gives an overview:
 
 Language | Dependencies to Generate Source | (Relative) Output Directory
 -------- | ------------------------------- | ---------------------------
-C#       | -                               | csharp
+C#       |                                 | csharp
 D        | -                               | dlang
 Golang   | -                               | go
-Java     | JNI headers                     | org/tinyspline
+Java     | Java Development Kit            | org/tinyspline
 Lua      | Lua headers                     | lua
-PHP      | Zend headers                    | php
+PHP      | PHP (Zend) headers*             | php
 Python   | Python headers                  | python
 Ruby     | Ruby headers                    | ruby
 
+* Please note that macOS comes with PHP, but does not have the Zend headers.  It is
+recommended to use a package manager (such as Homebrew) to obtain the headers.
+
 To simplify the usage of the bindings, the generated source files will be compiled and/or
 packaged if necessary. That is, for instance, the generated Java files will be compiled to
 .class files and packaged into a single jar archive. Thus, the following tools are
 required if you want to package the corresponding binding:
 
 Language | Required Tool(s)                 | Output File
 -------- | -------------------------------- | ----------------
-C#       | Any of: csc, mcs, dmcs, gmcs     | TinySpline.dll
+C#       | Any of: csc, mcs, dmcs, gmcs*    | TinySpline.dll
 Java     | javac and jar (available in JDK) | tinyspline.jar
 
-The remaining bindings do not need to be packaged (usually the script languages).
+* Mono support is yet to be implemented.  Sorry, macOS and Linux users!
```

Comment:
Actually, using Mono to compile the C# module is working very well on Linux.

Reply:
Oh really?  I didn't actually notice any mention of Mono in the CMake file, thanks for letting me know!

Reply:
As you can see [here](https://github.com/msteinbeck/tinyspline/blob/master/library/CMakeLists.txt#L622), CMake tries to find csc, mcs, dmcs, or gmcs. mcs, dmcs, and gmcs are mono specific compiler suites: http://www.mono-project.com/docs/about-mono/languages/csharp/

Reply:
Oh, that's right, `mcs` is Mono's compiler, I forgot -- whoops!  _I'm not too experienced with C#... XD_